packages/core/test/insured-bridge/BridgeAdmin.js

```diff
@@ -58,7 +61,11 @@ describe("BridgeAdmin", () => {
       defaultIdentifier
     ).send({ from: owner });
 
-    depositBox = await BridgeDepositBox.new(bridgePoolFactoryImpersonator, defaultBridgingDelay, ZERO_ADDRESS).send({
+    bridgePool = await BridgePool.new(bridgeAdmin.options.address, l1Token, timer.options.address).send({
```

Comment:
rather than mocking the `bridgePool` with a `bridgePoolFactoryImpersonator` as an EOA, deploy an actual contract to enable validation on whitelisting.